pkg/k8shandler/kibana/route.go

```diff
@@ -133,26 +130,19 @@ func (clusterRequest *KibanaRequest) createOrUpdateKibanaRoute() error {
 	return nil
 }
 
-func (clusterRequest *KibanaRequest) createOrUpdateKibanaConsoleLinks() error {
+func (clusterRequest *KibanaRequest) createOrUpdateKibanaConsoleLink() error {
```

Comment:
can we take the opportunity to rename this to somethign like "ReconcileXXXXX"

Reply:
Let's not start renaming here. The whole reconciler is full of `createOrUpdate..`.